src/postgraphql/graphiql/graphiql.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { render } from 'react-dom'
+import GraphiQL from 'graphiql'
+import { buildClientSchema, introspectionQuery } from 'graphql'
+import 'graphiql/graphiql.css'
+
+class EnhancedGraphiQL extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      schema: undefined,
+      token: undefined,
+    }
+  }
+
+  componentWillMount() {
+    setupStream({
+      onChange: this.onChange,
+      onOpen: this.onOpen,
+      onError: this.onError,
+    })
+  }
+
+  render() {
+    return (
+      <GraphiQL schema={this.state.schema} fetcher={this.fetcher}>
+        <GraphiQL.Logo>PostGraphQL GraphiQL</GraphiQL.Logo>
```

Comment:
Could we just call this GraphiQL still? I’d like to keep PostGraphQL branding to a minimum for now. If we wanted, we could add a `--graphiql-logo` option to customize it and allow users to whitelabel.